spring-cloud-skipper-docs/src/main/asciidoc/appendix-building.adoc

```diff
@@ -97,10 +89,11 @@ spring:
 
 Go to your command line, and from the /spring-cloud-skipper/ directory, run this command:
 ```
-`./mvnw clean package -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip` 
+./mvnw clean package -DskipTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip
 ```
-Which will build the jar, without running tests, creating documentation or checkstyle. 
+Which will build the jar, without running tests, creating documentation or checkstyle.
 
-Run `java -jar spring-cloud-skipper-server/target/spring-cloud-skipper-server-1.0.0.BUILD-SNAPSHOT.jar` if you updated the application.yml. If you created your own config, point to it by using: 
+Run `java -jar spring-cloud-skipper-server/target/spring-cloud-skipper-server-1.0.0.BUILD-SNAPSHOT.jar` if you updated the application.yml. If you created your own config, point to it by using:
```

Comment:
Btw, this is not only place in docs where reference to a build version is made. We should always bring in version from properties. Then it's always aligned with current one.

Reply:
Essentially something like
```
[source,bash,subs=attributes]
spring-cloud-skipper-server-{project-version}.jar
```